src/main/java/com/bitwig/dawproject/Lane.java

```diff
@@ -0,0 +1,5 @@
+package com.bitwig.dawproject;
+
+public class Lane extends Referenceable
```

Comment:
Optional: Base class for Track and Channel is fine conceptually but the name does not fit for Channel. Maybe "TrackOrChannel", "StructureItem" or "StructureComponent"?